tsdef/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended"
```

Comment:
Interesting! We use tslint in voice. Didn't know there was a typescript add-on for eslint. Did you try out tslint and find the eslint plugin works better?

Reply:
I chose eslint since it was already part of the project. I believe that the creators of tslint will be deprecating it in the future and that they're focusing their efforts on the plugin right now. [Reference](https://medium.com/palantir/tslint-in-2019-1a144c2317a9)